lib/measured/measurable.rb

```diff
@@ -28,11 +30,11 @@ def convert_to(new_unit)
   end
 
   def to_s
-    [value.to_f.to_s.gsub(/\.0\Z/, ""), unit].join(" ")
+    @to_s ||= "#{value_string} #{unit}"
   end
 
   def inspect
-    "#<#{ self.class }: #{ value } #{ unit }>"
+    @inspect ||= "#<#{self.class}: #{value_string} #{unit}>"
```

Comment:
I think you can just do this now:
`@inspect ||= "#<#{self.class}: #{self}>"` - It should call `to_s`